geograypher/meshes/derived_meshes.py

```diff
@@ -760,17 +812,39 @@ def get_single_pytorch3d_camera(self, camera: PhotogrammetryCamera):
             ),
         )
 
+        # Get distortion_params from each camera
+        distortion_params = camera.distortion_params
+
+        # Compute angular coefficients to convert from "ratio" to "angular" convention as expected by P3D.
+        # Metashape does not estimate higher-order coefficients if it doesn't have enough data to do so properly.
+        # In those cases, the missing parameters should be set to 0.
+        angular_coefficients = self.compute_angular_coeficients(
+            k1=distortion_params.get("k1", 0),
```

Comment:
If all `k` values are 0, does that mean the camera has no distortion and it reverts to being a pinhole model? Or does it collapse all the values to 0 or something? I just wanted to double check that 0 is the right default value.